benchmark/src/lib.rs

```diff
@@ -370,9 +370,10 @@ impl Benchmarker {
         &mut self,
         txn_reqs: &[SubmitTransactionRequest],
         senders: &mut [AccountData],
-    ) -> (f64, f64) {
+        submit_rate: Option<u64>,
+    ) -> (usize, usize, f64, f64) {
```

Comment:
The return value here becomes more and more cryptic.
Can you make a small struct with all these values, so it's nice and typed

Reply:
Sure, wrapped these values into a small struct.

> 1. Can you call it throughput or something similar, so it's easy to find and understand what this binary's supposed to do

Ok. I guess `LiSar` is not a very amusable name. Switch to `max_throughput`. 

> 
> ```
> ./target/release/lisar -f faucet_for_ruben -s temp_config \
> -n 32 -c 8 -r 2 -e 3 -m localhost:45345 -t pairwise \
> -u 800 -l 50 -i 5 -b 2
> ```
> 
> There are 12 parameters. Do different parameters affect the result throughput?

Following arguments are deprecated and the binary just uses its default values
* number of new accounts each time when we increase rate = 32
* number of repeated rounds when generate TXNs = 2
* number of clients = number of cpus on the machine
* TXN pattern = pairwise
* upper bound 8000 per client
After setting these default values, reusing RubenOpt inside `max_throughtput` looks very ugly. I will stack a new commit that introduce something like `CommonOpt` so the both binaries can share args in it.